build.gradle

```diff
@@ -34,6 +34,18 @@ configurations.all {
     transitive = true
 }
 
+task checkAll(type: GradleBuild) {
+    tasks = ['pmd', 'checkstyle', 'lintDebug', 'spotbugsDebug', 'testDebugUnitTest', 'connectedDebugAndroidTest']
+}
+
+task checkCode(type: GradleBuild) {
+    tasks = ['pmd', 'checkstyle', 'lintDebug', 'spotbugsDebug']
+}
+
+task checkTest(type: GradleBuild) {
```

Comment:
Plural form `checkTests` would be better.

Reply:
Do you mean rename this to `checkTests` ?

Reply:
yes